frontend/app/routes/accounts/show.js

```diff
@@ -1,6 +1,13 @@
 import BaseRoute from "../base";
+import { inject as service } from "@ember/service";
 
 export default class AccountShowRoute extends BaseRoute {
+  @service navService;
+
+  afterModel() {
+    this.navService.clear();
```

Comment:
Don't we want to have the corresponding Team/Folder of the shown Account selected in the SideNavBar? As a kind of Breadcrumb.